src/common/secured-property.ts

```diff
@@ -60,11 +60,11 @@ export function SecuredPropertyList<T>(
 ) {
   @ObjectType({ isAbstract: true, implements: [Readable, Editable] })
   abstract class SecuredPropertyListClass
-    implements Readable, Editable, Secured<T[]> {
+    implements Readable, Editable, Secured<T[] | null> {
     @Field(() => [valueClass], {
       nullable: options.isOverride,
     })
-    readonly value: T[];
+    readonly value: T[] | null;
```

Comment:
I don't think this change should be made

Reply:
In this case, how can we assign null to `ScriptureReferencesOverride.value`. Its type is `scriptrueRange[]`, so `can not assign null` error is occurring.

Reply:
Bleh. You are right. Ok let me make a change that makes it only nullable if that option is true.

Reply:
Ok pushed. Let me know if it doesn't work.

Reply:
Works like a charm.